domain/consensus/processes/pruningproofmanager/pruningproofmanager.go

```diff
@@ -607,13 +608,15 @@ func (ppm *pruningProofManager) dagProcesses(
 	return reachabilityManagers, dagTopologyManagers, ghostdagManagers
 }
 
-func (ppm *pruningProofManager) ApplyPruningPointProof(stagingArea *model.StagingArea, pruningPointProof *externalapi.PruningPointProof) error {
+func (ppm *pruningProofManager) ApplyPruningPointProof(pruningPointProof *externalapi.PruningPointProof) error {
```

Comment:
Can you add a comment that this function is not atomic and if it fails the outside scope should probably delete the database?